.travis.yml

```diff
@@ -1,26 +1,11 @@
-go_import_path: github.com/container-storage-interface/spec
+# default to using the container infrastructure
+sudo: false
 
-language: go
+# use the newer travis-ci build templates
+dist: trusty
 
-go:
-  - 1.8.x
+# selecting c as the language keeps the container to a minimum footprint
+language: c
 
-before_install:
-  - gem install gist
-  - curl -LO https://github.com/google/protobuf/releases/download/v3.3.0/protoc-3.3.0-linux-x86_64.zip
-  - unzip protoc-3.3.0-linux-x86_64.zip
-  - chmod +x bin/protoc
-  - export PATH="$(pwd)/bin:$PATH"
-
-install:
-  - go get -u github.com/golang/protobuf/proto
-  - go get -u github.com/golang/protobuf/protoc-gen-go
-  - go get -u google.golang.org/grpc
-  - CSI_SPEC_FILE=spec.md make csi.proto
-
-script:
-  - make csi.pb.go
-
-after_success:
-  - printf "%s" "$GITHUB_API_TOKEN" > "${HOME}/.gist"
-  - gist -d "https://travis-ci.org/container-storage-interface/spec/jobs/${TRAVIS_BUILD_ID}" *.{proto,go}
+# run make
+script: make
```

Comment:
Hi @jieyu,

You can then review them in tandem if you wish or I'll put them into the same PR. Your call. I don't want to reintroduce a lot of what I already pulled out.